src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString.split(';');
```

Comment:
The variable name 'splittedString' could be improved for better readability. Consider a name that reflects the content of the variable, such as 'styleDeclarations' or 'cssDeclarations', to indicate that it holds an array of individual CSS declarations.